Auth0Tests/AuthenticationSpec.swift

```diff
@@ -1173,7 +1173,7 @@ class AuthenticationSpec: QuickSpec {
                 
                 waitUntil { done in
                     auth.jwks().start {
-                        expect($0).to(beFailure())
+                        expect($0).to(beUnsuccessful())
```

Comment:
This is necessary because Nimble now provides a method called `beFailure` that conflicts with this one.